geode-junit/src/main/java/org/apache/geode/internal/jta/CacheUtils.java

```diff
@@ -144,12 +144,13 @@ public static void listTableData(String tableName) throws NamingException, SQLEx
     String sql = "select * from " + tableName;
 
     Connection conn = ds.getConnection();
-    Statement sm = conn.createStatement();
-    ResultSet rs = sm.executeQuery(sql);
-    while (rs.next()) {
-      System.out.println("id " + rs.getString(1) + " name " + rs.getString(2));
+    try (Statement sm = conn.createStatement()) {
+      ResultSet rs = sm.executeQuery(sql);
```

Comment:
`ResultSet` is also `AutoCloseable`, perhaps take the opportunity to clean this one up too.

Reply:
the analyzer did not detect it because there was a rs.close at the end of the block.

Reply:
same with `Connection`. If there is a .close() at the end of the block it does not consider it as a violation.